src/components/Badges/index.js

```diff
@@ -1,18 +1,30 @@
 import VerifiedBadge from '../VerifiedBadge';
 
-function Badges({ tokenId, verified }) {
+function Badges({ content }) {
   return (
-    <>
-      <VerifiedBadge
-        color={!verified ? 'greyLight' : 'primary'}
-        verified={verified}
-        badgeName={!verified ? 'Waiting for verification' : 'Verified'}
-      />
-      <VerifiedBadge
-        color="secondary"
-        badgeName={!tokenId ? 'Token not issued' : 'Token Issued'}
-      />
-    </>
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    (<>
+      {content?.map((data, index) => {
+        const {
+          color,
+          verified,
+          badgeName,
+          onClick = null,
+          disabled = false,
+        } = data;
+        return (
+          <VerifiedBadge
+            // eslint-disable-next-line react/no-array-index-key
+            key={index}
+            color={color}
+            verified={verified}
```

Comment:
verified isn't a language that the general component should know, what if we want to build a badge for `matched` rather than `verified` ? with color and icon is good enough, or we can use icon name like: leading-icon='verified', 'matched', 'null'